test/resources/api_key.js

```diff
@@ -17,26 +17,26 @@ describe('ApiKey Resource', () => {
     expect(apiKey).to.be.a('function');
   });
 
-  it('throws on delete', () => ApiKey.delete('id').then(() => {}, err => {
+  it('throws on delete', () => ApiKey.delete('id').then(() => { }, err => {
     expect(err).to.be.an.instanceOf(NotImplementedError);
   }));
```

Comment:
A small problem I see (which this PR doesn't need to address) is:

Won't test like this one pass if the Promise **doesn't** reject, too? Probably not urgent, but seems fishy to me.

Reply:
The whole test suite in this library is awful. We have an internal ticket to overhaul it completely which I'll be doing next week (most of the issues with testing should get addressed there). Thanks for calling this out!